application/src/main/scala/com/azavea/franklin/api/services/TileService.scala

```diff
@@ -87,6 +98,15 @@ class TileService[F[_]: Sync: LiftIO](
                 .bytes
             }
           }
+          case Valid(mbt: MultibandTile) if mbt.bandCount == 1 =>
+            val cmap = rs.tiff.options.colorMap getOrElse {
+              val greyscaleRamp = greyscale(255)
+              val hist          = histograms(0)
+              val breaks        = hist.quantileBreaks(100)
+              greyscaleRamp.toColorMap(Vector(breaks.min, breaks.max))
```

Comment:
Note that I don't really have any idea whether this does an ok job interpolating between the min and max

Reply:
nevermind, it didn't, so I fixed it